src/courses/templates/courses/chart_tags_progress.html

```diff
@@ -0,0 +1,157 @@
+<script>
```

Comment:
Can you update the styling to match the other chart/histogram?  Use one gray for quests and the other gray for badges, and then maybe no transparency, so the divisions between different quests are visible and not confused with the gridlines.

Reply:
We can remove the Total XP bar, that's redundant, right?  Already have that info from the other two stacked together?

Reply:
done
done

Reply:
"then maybe no transparency, so the divisions between different quests are visible and not confused with the gridlines.'  Can still see gridlines through the bars

Reply:
oops fixed